proposals/0759-react-native-frameworks.md

```diff
@@ -0,0 +1,452 @@
+  - Error and Crash reporting
+- **Native build integration** - The React Native package is responsible for defining how the Android & iOS final artifact are built, as well as how Java/Kotlin/Swift/ObjC/ObjC++ and C++ should be compiled and distributed.
+- **3rd party libraries** - The React Native package is responsible for defining integration API for 3rd party libraries (i.e. how to create libraries for React Native and how to integrate them with the runtime). Frameworks can build on top of those APIs to simplify the library integrations and discovery via mechanisms such as auto-linking.
+
+### What does the React Native package not do
+
+React Native Frameworks provide features that **most native apps** that are intending to reach production eventually need.
+
+Here is a (non-exhaustive) list of areas of what the React Native package is **not directly focusing** on, and is delegating this responsibility to frameworks:
+
+- **Project Scaffolding** - Frameworks can offer tools and templates to create new projects so that users can get started developing in a matter of seconds.
+- **Tooling for React Native upgrade DevX** - Similarly to project scaffolding, framework can offer tools to make sure the project can easily keep up with the latest React Native version. Tools such as the upgrade helper, Expo Config Plugins and others fit in this category.
+- **Navigation and Routing solutions** - React Native doesn’t offer a navigation or a routing solution out of the box. Frameworks can either offer their own solution or recommend a preferred navigation solution to use.
+- **Managing the build experience** - Framework can offer tools to simplify the native builds and move to fully managed native builds that are happening in the cloud.
+- **Manage the publishing experience** - Frameworks can offer solutions to publish applications to public stores (App Store/Play Store and others) either locally or managed in the cloud.
+- **Providing OTA updates** - Frameworks can offer mechanisms to deploy Over-the-air updates to users. Tools such as EAS Update & Codepush fall into this category.
+- **Auto-Linking libraries** - Frameworks can offer mechanisms to simplify linking of external 3rd party libraries.
+- **Support new targets via Out-of-tree Platforms** - Frameworks can offer support for additional targets by using OOT Platforms, in order to offer a more cohesive Development experience.
+
+## What do we recommend to React Native app developers?
+
+As mentioned before, we believe React Native Framework offers all the tools you’ll eventually need to get to production quickly while offering a superior developer experience. As we mentioned before:
+
+> You’re either using a React Native Framework or you’re building your own React Native Framework
+
+We believe that an existing React Native Framework should be the default solution for the **majority of app developers**, specifically for beginners and developers which lack a native background.
```

Comment:
```suggestion
We believe that an existing React Native Framework should be the default solution for the **majority of app developers**, including developers who have more of a web background than a native background.
```
I find this wording to be more accurate and welcoming, in a way. General software engineering experience and native development skills both are valuable with a Framework, and I suspect a good part of the audience has a web background, coming from React DOM.